opensrp-chw/src/main/java/org/smartregister/chw/reporting/ChwReport.java

```diff
@@ -140,22 +142,25 @@ public static void createAncReportViews(ViewGroup mainLayout, List<Map<String, I
     public static void createChildReportViews(ViewGroup mainLayout, List<Map<String, IndicatorTally>> indicatorTallies) {
         //Disclaimer: Pie charts have binary slices yes and no with different tallying done separately ;)
 
-        PieChartSlice indicator7_1 = ReportingUtil.getPieChartSlice(ReportContract.IndicatorView.CountType.LATEST_COUNT, ReportingConstants.ChildIndicatorKeys.COUNT_OF_CHILDREN_0_59_WITH_BIRTH_CERT, mainLayout.getContext().getResources().getString(R.string.yes), mainLayout.getContext().getResources().getColor(R.color.pie_chart_yes_green), indicatorTallies);
-        PieChartSlice indicator7_2 = ReportingUtil.getPieChartSlice(ReportContract.IndicatorView.CountType.LATEST_COUNT, ReportingConstants.ChildIndicatorKeys.COUNT_OF_CHILDREN_0_59_WITH_NO_BIRTH_CERT, mainLayout.getContext().getResources().getString(R.string.no), mainLayout.getContext().getResources().getColor(R.color.pie_chart_no_red), indicatorTallies);
-        appendView(mainLayout, new PieChartIndicatorView(mainLayout.getContext(), ReportingUtil.getPieChartDisplayModel(ReportingUtil.addPieChartSlices(indicator7_1, indicator7_2), R.string.children_0_59_months_with_birth_certificate, null)));
+        if (!Constants.LMH_FLAVOUR.equals(BuildConfig.FLAVOR)) {
```

Comment:
Introduce a feature on the flavor (`org.smartregister.chw.application.ChwApplication.Flavor`) instead of hard coding the build flavor. This would help is another flavor is added and has similar requirements

Reply:
Done!